metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -55,20 +62,28 @@
     private static final String DOI_FIRST_PART = "10";
     private static final String HTTPS = "https";
     private static final String HTTP = "http";
-    public static final ValueFactory valueFactory = SimpleValueFactory.getInstance();
-    //Todo: this URI should come from a Config/Environment. How is that done in NVA
-    public static final String API_URI_NVA_PUBLICATION_CHANNELS = "https://api.dev.nva.aws.unit.no/publication-channels";
     private final HttpClient httpClient;
     private final TranslatorService translatorService;
     private final Repository db = new SailRepository(new MemoryStore());
+    private final URI publicationChannelsHostUri;
 
-    public MetadataService() throws IOException {
-        this(getDefaultHttpClient());
+    public MetadataService() {
+        this(getDefaultHttpClient(), defaultPublicationChannelsHostUri());
     }
 
+    /**
+     * @deprecated  For testing, we should also inject the URI so that we can use WireMock.
+     * @param httpClient the HttpClient.
+     */
+    @Deprecated
     public MetadataService(HttpClient httpClient) {
+        this(httpClient, defaultPublicationChannelsHostUri());
+    }
+
+    public MetadataService(HttpClient httpClient, URI publicationChannelsHostUri) {
```

Comment:
Injecting the publicationChannelsURI allows us to use Wiremock.

Reply:
I think this Wiremock smells a bit like over-engineering